invenio_app_ils/records_relations/api.py

```diff
@@ -200,6 +200,7 @@ def get(self):
                 r = self._build_parent(child_pid, name)
 
                 relations.setdefault(name, [])
+                r["relation_type"] = name
```

Comment:
can you please add this in the `_build_parent` and other methods (the ones below) instead?